README.md

```diff
@@ -16,6 +16,22 @@ It's possible to implement a custom handling of incoming messages, but the defau
 To implement your own custom handling of received messages, override the operation `TCPServer::handleReceivedMessageCustom()` and set the config property `defaultHandlingOfReceivedMessages` to false.
 
 ## Build the Library
+
+### Master branch
```

Comment:
Suggested change:
The original implementation of the library uses the POCO library and requires Model RealTime 10.3 2018.48 or later. Checkout the tag "poco" if you want to use this version.

The latest implementation, that is available on the master branch, has no dependency on POCO, but instead requires Model RealTime 12.0.2 or later.